sdk/finding.ts

```diff
@@ -67,7 +78,9 @@ export class Finding {
     type,
     metadata = {},
     addresses = [],
-    labels = []
+    labels = [],
+    uniqueKey = '',
+    chainId = -1,
```

Comment:
not sure we should be setting these default values unless we want them being set in the backend even when a user does not specify them. i think we should leave them as undefined